skiko/src/commonMain/kotlin/org/jetbrains/skia/ColorSpace.kt

```diff
@@ -12,10 +12,35 @@ class ColorSpace : Managed {
 
         val sRGB = ColorSpace(_nMakeSRGB(), false)
         val sRGBLinear = ColorSpace(_nMakeSRGBLinear(), false)
-        val displayP3 = ColorSpace(_nMakeDisplayP3(), false)
+        val displayP3: ColorSpace = makeRGB(TransferFunction.sRGB, Matrix33.displayP3ToXYZD50)!!
+
+        /**
+         * Creates a color space from a transfer function and a 3x3 transformation to XYZ D50.
+         * Matrix33 offers various factories for obtaining a toXYZD50 matrix.
+         */
+        fun makeRGB(transferFunction: TransferFunction, toXYZD50: Matrix33): ColorSpace? {
```

Comment:
`internal` visibility should work for tests. Should it be public API?

Reply:
``ColorSpace::makeRGB`` is the key API added by this PR: it allows to create color spaces with arbitrary transfer functions and gamuts. So it absolutely has to be public.